scripts/jenkins/qa_openstack.sh

```diff
@@ -321,8 +321,14 @@ esac
 
 glance image-list
 imgid=$(glance image-list|grep debian-5|cut -f2 -d" ")
-mkdir -p ~/.ssh
-( umask 77 ; nova keypair-add testkey > ~/.ssh/id_rsa )
+
+if [ -f /root/.ssh/id_rsa.pub ]; then
```

Comment:
`qa_openstack.sh` is run always from root user, but I've changed it back as you wish.